faunadb-java/src/test/java/com/faunadb/client/ObjectCodecSpec.java

```diff
@@ -32,6 +32,7 @@
 
   @Test
   public void testSimpleMap() {
+    System.out.println("HI");
```

Comment:
funny story. JUnit by default squelches stdout. Until I added the +v flag to it. So I had no idea that half of these printlns were in there